src/fragments/seo.js

```diff
@@ -1,20 +1,29 @@
 import { graphql } from "gatsby"
 
 export const SEO = graphql`
-  fragment SEO on MarkdownRemarkFrontmatterSeo {
+  fragment SEO on SeoMeta {
```

Comment:
This was a recent addition to gatsby-remark-ample in that it creates child nodes for all structured content so that we can have both structured content and still use a fragment.